<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ракета</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QSplitter" name="control">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="F">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>158</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>158</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Сила тяги</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="F_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>278</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="m0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt  &quot;MS Shell Dlg 2&quot; ;</string>
         </property>
         <property name="text">
          <string>Масса груза и топлива</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m0_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>кг</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="mk">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Масса груза</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="mk_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>255</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>кг</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="alpha">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>189</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Расход топлива</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="alpha_input">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>218</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>218</width>
           <height>55</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>46</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>46</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>кг/с</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="vt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>227</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>227</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Построить график v(t)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ht">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Построить график h(t)</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="Graph"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
